src/Features/Core/Portable/CodeRefactorings/SyncNamespace/AbstractChangeNamespaceService.cs

```diff
@@ -47,56 +52,81 @@ internal abstract class AbstractChangeNamespaceService<TNamespaceDeclarationSynt
     {
         private static readonly char[] s_dotSeparator = new[] { '.' };
 
+        /// <summary>
+        /// The annotation used to track applicable container in each document to be fixed.
+        /// </summary>
+        protected static SyntaxAnnotation[] ContainerAnnotation { get; } = new[] { new SyntaxAnnotation() };
```

Comment:
1.  Should be plural.  
2. Don't have mutable top level arrays.  Someone might change it out from underneat you.